### ISSUE TITLE
NaN loss in first epoch

### ISSUE THREAD

**Issue by PhenomenalOnee** [JinghuiZhou/awesome_face_antispoofing#23]
when choosing a different network like  MyseNet1_0 for training loss becomes nan.
Also when i try to modified the resnet18 architecture training loss becomes nan.
please see the screenshot below:
(https://user-images.githubusercontent.com/47495143/66979488-56fd9f00-f0cb-11e9-974b-9c96cc18bb21.png)

**Comment by PhenomenalOnee** [comment]
Got the solution..
Need to add the EXTRA BatchNorm2d layer in MyseNet1_0 that helps in normalizing the output from conv1 layer.
https://user-images.githubusercontent.com/47495143/66995141-20844c00-f0ec-11e9-8738-ab58cf0d33b7.png)
